css/style.css

```diff
@@ -15,7 +15,6 @@ margin:auto;}
 
   .header{
     display: flex;
-    
     height: 15vh;
   }
 
```

Comment:
Please you may use display: grid for the main container, this makes the layout easy to read especially by search engines